sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/aio/_path_client_async.py

```diff
@@ -650,7 +652,8 @@ async def _set_access_control_internal(
                             name=failure.name,
                             is_directory=failure.type == 'DIRECTORY',
                             error_message=failure.error_message) for failure in resp.failed_entries],
-                        continuation=last_continuation_token))
+                        continuation=last_continuation_token  # type: ignore [arg-type]
```

Comment:
I think you fixed this in sync, can you do the same for async? There is an ignore below too